app/models/notice.rb

```diff
@@ -104,13 +104,13 @@ def tags_string
 
   def self.weekly_watched
     Notice.select('notices.*')
-      .joins(:activities).merge(
-        Activity
-        .select('notice_id, count(activities.id) AS count_id')
-        .where('activities.created_at >= ?', 1.week.ago)
-        .group(:notice_id)
-      )
-      .order('count_id DESC')
+          .joins(:activities).merge(
+            Activity
+            .select('notice_id, count(activities.id) AS count_id')
+            .where('activities.created_at >= ?', 1.week.ago)
+            .group(:notice_id)
+          )
+          .order('count_id DESC')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.